test/integration/emitter_test.cpp

```diff
@@ -382,6 +382,20 @@ TEST_F(EmitterTest, ScalarFormat) {
       "crazy\tsymbols that we like");
 }
 
+TEST_F(EmitterTest, LiteralWithoutTrailingSpaces) {
+  out << YAML::BeginMap;
+  out << YAML::Key << "key";
+  out << YAML::Literal;
```

Comment:
This should have 'YAML::Value' first. I don't think it makes an output difference, though.

Reply:
> This should have 'YAML::Value' first. I don't think it makes an output difference, though.
:see_no_evil: sure, thanks.